drizzlepac/haputils/config_utils.py

```diff
@@ -167,7 +168,8 @@ def _determine_conditions(self, prod_obj):
                         if n_exp >= 6:
                             self.conditions.append("acs_wfc_any_n6")
                     else:
-                        sys.exit("INVALID ACS DETECTOR!")
+                        log.error("INVALID ACS DETECTOR!")
```

Comment:
For messages like this, I have been trying to print the information the program currently has in memory in case there is something amiss.  I suggest printing out the value of the variable (e.g., self.detector).  This comment should be applied to other similar statements.

Reply:
Issue resolved by commit [fafc9d2](https://github.com/spacetelescope/drizzlepac/pull/1267/commits/fafc9d20e0626b42eb8c6356308f374d1cf93761)